src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Windows/MyExtensionsApp.Windows.csproj

```diff
@@ -177,18 +177,4 @@
 	</ItemGroup>
 
 	<Import Project="..\MyExtensionsApp.Base\base.props" />
-
-	<ItemGroup>
-		<!--
-		If you encounter this error message:
-
-			error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll. Please update to a newer .NET SDK in order to reference this assembly.
-
-		This means that the two packages below must be aligned with the "build" version number of
-		the "Microsoft.Windows.SDK.BuildTools" package above, and the "revision" version number
-		must be the highest found in https://www.nuget.org/packages/Microsoft.Windows.SDK.NET.Ref.
-		-->
-		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22621.28" />
-		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22621.28" />
-	</ItemGroup>
```

Comment:
I just noticed this is in a templates. IIRC, Jerome preferred to keep this in Uno templates.

Reply:
Not required since we're referencing latest buildtools - different for Uno core that takes dependency on earlier version where there are known issues